roles/inject_openshift_kube_objects/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "Create the kube_objects directory"
+  shell: "mkdir -p {{ kube_objects_dir }}"
+
+- name: "Get the CVP private key"
+  copy:
+    src: "{{ ssh_private_key }}"
+    dest: "{{ kube_objects_dir }}/private_key"
+    mode: 0600
+  no_log: true  
```

Comment:
just a question. Could you explain what is a benefit of this copy?
IMHO it is simpler to directly use shorter version `{{ ssh_private_key }}` instead of `"{{ kube_objects_dir }}/private_key`.

Reply:
oh was more like a copy/paste & leftover